test/ci/backend/test_lava.py

```diff
@@ -1257,3 +1258,21 @@ def test_resubmit_job_lava_error(self, __resubmit__):
             job_id="1236"
         )
         self.assertRaises(TemporarySubmissionIssue, lava.resubmit, testjob)
+
+    def test_resubmit_job_lava_4xx_reply(self):
+        self.backend.url = "http://example.com/api/v0.2/"
+        lava = self.backend.get_implementation()
+        test_definition = "foo: 1\njob_name: bar"
+        testjob = TestJob(
+            definition=test_definition,
+            backend=self.backend,
+            target=self.project,
+            submitted=True,
+            job_status='Canceled',
+            job_id="1236"
+        )
+        with requests_mock.Mocker() as m:
+            m.post("http://example.com/api/v0.2/jobs/1236/resubmit",
```

Comment:
I saw this error: `requests_mock.exceptions.NoMockAddress: No mock address: POST http://example.com/api/v0.2/jobs/1236/resubmit/`

Maybe `requests_mock` is adding the extra slash there? And it seems the requested URL and the mocked one need to be exactly the same: https://stackoverflow.com/questions/62981932/requests-mock-nomockaddress-exception

Reply:
it's my mistake. I forgot to update the address when adding a trailing slash into the code.